modules/installation-configuration-parameters.adoc

```diff
@@ -869,6 +869,10 @@ Additional {rh-openstack} configuration parameters are described in the followin
 |For compute machines, the root volume's type. This property is deprecated and is replaced by `compute.platform.openstack.rootVolume.types`.
 |String, for example, `performance`. ^[2]^
 
+|`compute.platform.openstack.rootVolume.zones`
+|For compute machines, the Cinder availability zone to install root volumes on. If you do not set a value for this parameter, the installation program selects the default availability zone. This parameter is mandatory when `compute.platform.openstack.zones` and `compute.platform.openstack.rootVolume.types` are defined.
```

Comment:
```
and `compute.platform.openstack.rootVolume.types` 
```

This is not useful, the type(s) parameter is required anyway.

Reply:
sure. Will remove that. As per my Jira comment, I had added this since it was mentioned in the ticket description.